Sources/Automerge/patch.swift

```diff
@@ -19,12 +27,19 @@ public struct Patch: Equatable {
     }
 }
 
+/// The type of change to apply along with the data to change.
 public enum PatchAction: Equatable {
+    /// Put a new value at the property for the identified object.
     case Put(ObjId, Prop, Value)
+    /// Insert a collection of values at the index you provide for the identified object.
     case Insert(ObjId, UInt64, [Value])
+    /// Splices characters into and/or removes characters from the identified object at a given position within it.
```

Comment:
I suppose here we should probably mention that the index is a utf-8 code point index and not a grapheme cluster index

Reply:
Definitely, I was guessing on the specifics of these internals (that's where I was reading the C API in the repo to get the details) - I'll make updates accordingly and make sure that's detailed here.